integration-tests/src/tests/references.test.ts

```diff
@@ -54,10 +54,11 @@ suite('Bitbake References Test Suite', () => {
   }).timeout(300000)
 
   test('References appear properly in Bash on variable', async () => {
-    const position = new vscode.Position(7, 13)
+    const position = new vscode.Position(9, 13)
     const referenceRanges = [
-      new vscode.Range(6, 4, 6, 7),
-      new vscode.Range(7, 12, 7, 15)
+      new vscode.Range(5, 0, 5, 3),
+      new vscode.Range(8, 4, 8, 7),
```

Comment:
Can you show me an example of a variable defined outside the bash function that can be referenced inside the bash function directly? (without ${})

Reply:
You mean "real life" examples?

This is the only one I could find: https://git.yoctoproject.org/poky/tree/meta/recipes-bsp/v86d/v86d_0.1.10.bb#n24

But I admit it is not obvious, so now I'm thinking I might be wrong.

Reply:
In this case, it makes sense.